### ISSUE TITLE
Need data loader for sentences

### ISSUE THREAD

**Issue by yogeshhk** [graph4ai/graph4nlp#341]
## 🚀 Feature

A class which will take sentences, populate dataset having graph topology built into it and also create data loader, ready to be used for inferencing

## Motivation

As discussed in issues #319 and #316 for doing inferencing we need data loader populated from text/sentences. Currently all data loaders are populated from files such as train and test. We can not give just sentences to any model.

Also, only F1 like metrics are made available. No actual predicted label is available,

## Alternatives
One can store given sentences in, say, test file and then use the functionality.

**Comment by AlanSwift** [comment]
This feature is under development now.
Please refer to this [example](https://github.com/graph4ai/graph4nlp/blob/develop/examples/pytorch/semantic_parsing/graph2seq/inference.py) to follow up the newest inference pipeline.
In this example, users can only use pretrained checkpoint and their own test cases to do inference.

However, what does "No actual predicted label is available" mean?

**Comment by AlanSwift** [comment]
Further explanation about the inference pipeline:
1. In the newest [demo](https://github.com/graph4ai/graph4nlp/blob/develop/examples/pytorch/semantic_parsing/graph2seq/inference.py), the users need to store their own test cases in a single file named "test.txt" in an [independent folder](https://github.com/graph4ai/graph4nlp/tree/develop/examples/pytorch/semantic_parsing/graph2seq/jobs_inference/raw) (Note that it is independent of the training data folder). Thus during inference, users only need to provide: 1. pretrained checkpoint, 2. their own test cases. 
2. Furthermore, we will provide a user-friendly inference API in Graph2X (only Graph2Seq and Graph2Tree currently) as follows:
`from graph4nlp.pytorch.models.graph2seq import Graph2Seq`
`model = Graph2Seq.load_checkpoint(CHECKPOINT_PATH, CHECKPOINT_NAME) # load a pretrained model`
`output = model.predict("I am a student.") # you only need to feed the raw inputs, no other scripts are needed.`



Does these two coming features meet your need?

**Comment by yogeshhk** [comment]
> This feature is under development now.
> Please refer to this [example](https://github.com/graph4ai/graph4nlp/blob/develop/examples/pytorch/semantic_parsing/graph2seq/inference.py) to follow up the newest inference pipeline.
> In this example, users can only use pretrained checkpoint and their own test cases to do inference.
> 
> However, what does "No actual predicted label is available" mean?

For NER, would like to have not tags but actually labels as out put. Eg.

```
model.predict("My name is Sylvain and I work at Hugging Face in Brooklyn.")

>>>
[{'entity_group': 'PER', 'score': 0.99816, 'word': 'Sylvain', 'start': 11, 'end': 18}, 
 {'entity_group': 'ORG', 'score': 0.97960, 'word': 'Hugging Face', 'start': 33, 'end': 45}, 
 {'entity_group': 'LOC', 'score': 0.99321, 'word': 'Brooklyn', 'start': 49, 'end': 57}
]
```
I did not find any way to get such output.

**Comment by AlanSwift** [comment]
Ok. I will involve @xguo7. She is responsible for this example.

**Comment by xguo7** [comment]
> > This feature is under development now.
> > Please refer to this [example](https://github.com/graph4ai/graph4nlp/blob/develop/examples/pytorch/semantic_parsing/graph2seq/inference.py) to follow up the newest inference pipeline.
> > In this example, users can only use pretrained checkpoint and their own test cases to do inference.
> > However, what does "No actual predicted label is available" mean?
> 
> For NER, would like to have not tags but actually labels as out put. Eg.
> 
> ```
> model.predict("My name is Sylvain and I work at Hugging Face in Brooklyn.")
> 
> >>>
> [{'entity_group': 'PER', 'score': 0.99816, 'word': 'Sylvain', 'start': 11, 'end': 18}, 
>  {'entity_group': 'ORG', 'score': 0.97960, 'word': 'Hugging Face', 'start': 33, 'end': 45}, 
>  {'entity_group': 'LOC', 'score': 0.99321, 'word': 'Brooklyn', 'start': 49, 'end': 57}
> ]
> ```
> 
> I did not find any way to get such output.

Hello,

Currently we do not implement the NER as a model function in the library, but an outside example code. If you would like to find ways to get the label from the tags in this example, you could refer to the file :    "/examples/pytorch/name_entity_recognition/conlleval.py". Thanks for your attention, we will consider adding the NER as a model function in the next version.

Best,
Xiaojie
